### ISSUE TITLE
Error when mapscript-python is not installed

### ISSUE THREAD

**Issue by fegyi001** [faunalia/rt_mapserver_exporter#7]
I just installed the plugin and wanted to use, but an error occured when the button was pushed (`ImportError: No module named mapscript`).
I think it would be nicer if the user should receive a more detailed message in a window.

My recommendation: modify the `run` method in plugin.py for example like this:

```
def run(self):
    try:
        import mapscript
        from .mapfileexportdlg import MapfileExportDlg
        dialog = MapfileExportDlg(self.iface, self.iface.mainWindow())
        dialog.show()
        dialog.exec_()
    except ImportError:
        QMessageBox.warning(None, "Import Error", "Missing module: 'mapscript'. Please install 'mapscript-python' first.")
```

It works as expected.

**Comment by luipir** [comment]
you are right, can you prepare a pull request? I'm really busy and simpler to me to merge and prepare package

**Comment by brncsk** [comment]
Luigi,
I have agreed with @fegyi001 to prepare a fix. I plan to do it in the following days if it's okay for both of you guys.

**Comment by fegyi001** [comment]
It is ok for me if Ádám fixes the problem. I suggested a solution above, but I will not feel offended if another fix will be implemented :)

**Comment by brncsk** [comment]
Ah ok, I missed that part, I'll have a look at it.

**Comment by brncsk** [comment]
PR is available at #8, thanks @fegyi001!

**Comment by luipir** [comment]
Hi I'll merge 
do you think it's enough to prepare a new plugin version? is a simple error managing
I would wait to collect more changes before to prepare a new version. What do you think about?
